plugins/linux-capture/xcompcap-main.cpp

```diff
@@ -318,7 +318,7 @@ void XCompcapMain::updateSettings(obs_data_t *settings)
 	}
 
 	if (p->win)
-		XSelectInput(xdisp, p->win, StructureNotifyMask | ExposureMask);
+		XSelectInput(xdisp, p->win, StructureNotifyMask | ExposureMask | VisibilityChangeMask);
```

Comment:
Yeah sorry, I applied the patch from the bugtracker without looking at that. It wasn't a line easy to wrap so I did the best I could to match as closely as possible to the surrounding style...